### ISSUE TITLE
 Error message for --add-chromosome-info 

### ISSUE THREAD

**Issue by kusimpkins** [htseq/htseq#90]
Hello,

I'm trying to run a script using HTSeq- count. I tried to use **--add-chromosome-info** as I need to have the chromosomal coordinate in my output file. Here is the script that I used:

**module load HTSeq

htseq-count -s yes -r pos -t exon -i gene_id --additional attr=gene_name - - additional-attr=exon_number  --add-chromosome-info -f bam <alignment_files> <gff_file>**

However, I keep getting this error message:
 _**usage: htseq-count [options] alignment_file gff_file
htseq-count: error: unrecognized arguments: --add-chromosome-info**_

If I remove the **--add-chromosome-info** in the script, it runs successfully. I didn't see any examples in the documentation. Am I implementing it incorrectly somehow? 

Thank you,

**Comment by iosonofabio** [comment]
Hi @kusimpkins ,

Never heard of that option. Where did you read about it? 

Cheers, 
Fabio

**Comment by kusimpkins** [comment]
Hello @iosonofabio 

I found it in your document page, under htseq-count: counting reads within features (https://htseq.readthedocs.io/en/master/htseqcount.html)

Thank you,

Best,

**Comment by iosonofabio** [comment]
Indeed. Perhaps a regression, adding this should not take long. Would you be able to make a PR anytime soon?

**Comment by kusimpkins** [comment]
I'm only familiar with this code as a user. Are you suggesting I fork it, search through the old version for this function and add it back to my fork then do a PR? If it is a fairly straightforward change I can imagine that (especially if it was implemented in the past), however I was hoping I just am not using the htseq options correctly. 

Is it possible that the option's name was changed, and it is part of the new version under a new name? I am not familiar with the development history in detail.

**Comment by kusimpkins** [comment]
(i.e. something like a binary search)

**Comment by iosonofabio** [comment]
roughly speaking, just not exactly. I can definitely fix this but also have a 1 month old baby at home and it might take a while until I get around fixing it in practice. Therefore, I'm trying to explore if you could try making a PR with a proposed fix so I could help reviewing it and so on without having to do all the coding myself.

The procedure would be:
1 fork the repo
2 implement the change on your machine and test it
3 push onto your remote fork and then make a PR
4 I'll review it and help you get it in

If that's something you are in a position to consider, that would just speed up the fix. Otherwise, please be patient and I'll get to this at some point later this year

Thanks!

**Comment by kusimpkins** [comment]
I appreciate you even responding with a one month old baby. Congratulations! I know how that goes as I have two myself. Anyways, I will ask my collaborator to look into this. I am under a very short deadline to finish up a paper so I was wondering if you know any other tools that you would recommend to add chromosomal information.

Thank you,
Best,

**Comment by iosonofabio** [comment]
pybiomart can give you the info in 3 seconds

**Comment by kusimpkins** [comment]
Thank you very much for your suggestion. I will try that.

Best,
